plugins/score-card/src/components/ScoreCard/ScoreCard.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { Chip, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import React, { useEffect } from 'react';
+import { useAsync } from 'react-use';
```

Comment:
These imports are the cause of the typescript issues. `react-use` is unmaintained and has outdated types, importing the entire package breaks type checking. You'll need to either switch the imports to `react-use/lib/useAsync` etc., or use `@react-hookz/web` instead

Reply:
Thanks! Hopefully fixed that.